### PR TITLE
Add support for Google Analytics and SEO

### LINKED ISSUE #47: SEO and Analytics in header
Hi,

As I mentioned in minima theme, there are couple useful things located in header like SEO and google analytics included, take a look: https://github.com/jekyll/minima/blob/master/_includes/head.html . Could you please also include these? It will be really helpful.

Thanks

### PR DESCRIPTION
**Pull request by prodingerd**
Resolves #47.

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -1,17 +1,8 @@
...
-  {% endif %}
 
-  <title>
-    {% if page.title == "Home" %}
-      {{ site.title }}
-    {% else %}
-      {{ page.title }} &middot; {{ site.title }}
-    {% endif %}
-  </title>
+  {% seo %}
```
Comment (chesterhow) [inline _includes/head.html]: May I know what this line does?

Comment (prodingerd) [reply _includes/head.html]: The [SEO tag](https://github.com/jekyll/jekyll-seo-tag/blob/master/docs/usage.md) for Jekyll will automatically generate some HTML elements such as `title`, `description`, `author` etc.

Below is an example of what `{% seo %}` would output for https://chesterhow.github.io/tale

```html
<!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Home | Tale</title>
<meta name="generator" content="Jekyll v3.7.4" />
<meta property="og:title" content="Home" />
<meta name="author" content="Chester How" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Minimal Jekyll theme for storytellers" />
<meta property="og:description" content="Minimal Jekyll theme for storytellers" />
<link rel="canonical" href="https://chesterhow.github.io/tale/" />
<meta property="og:url" content="https://chesterhow.github.io/tale/" />
<meta property="og:site_name" content="Tale" />
<link rel="next" href="https://chesterhow.github.io/tale/page2" />
<script type="application/ld+json">
  {
    "@type":"WebSite",
    "url":"https://chesterhow.github.io/tale/",
    "name":"Tale",
    "headline":"Home",
    "author":{"@type":"Person","name":"Chester How"},
    "description":"Minimal Jekyll theme for storytellers",
    "@context":"http://schema.org"
  }
</script>
<!-- End Jekyll SEO tag -->
```

Comment (prodingerd) [reply _includes/head.html]: Also, if you do not like the way `{% seo %}` outputs the title, the following line can be used to disable title generation: `{% seo title=false %}`.

Comment (chesterhow) [reply _includes/head.html]: I see.. that's pretty handy!

File: Gemfile [hunk 2]
```diff
@@ -2,3 +2,5 @@
 
 source "https://rubygems.org"
 gemspec
+
+gem 'jekyll-seo-tag'
```
Comment (chesterhow) [inline Gemfile]: For adding of new Gems, add it in the `tale.gemspec` file instead. Also add it in the `_config.yml` file under the `plugins` section

Comment (prodingerd) [reply Gemfile]: Done!

**Comment by chesterhow** [comment]
Looks good to me, thanks again for your contributions! Hopefully you signed up for Hacktoberfest 😉

**Pull request merged**